internal/statistics.go

```diff
@@ -140,38 +139,30 @@ func (s *statistics) sendStatsToOwner(stats *bytes.Buffer) {
 	}
 }
 
-func (s *statistics) registerMetrics() {
+func (s *statistics) collectMetrics() {
+	s.collectOSMetrics()
+	s.collectRuntimeMetrics()
+}
+
+func (s *statistics) collectRuntimeMetrics() {
 	var m runtime.MemStats
 	runtime.ReadMemStats(&m)
+	s.registerGauge("runtime.availableProcessors", strconv.Itoa(runtime.NumCPU()))
+	s.registerGauge("runtime.freeMemory", strconv.Itoa(int(m.HeapIdle)))
+	s.registerGauge("runtime.totalMemory", strconv.Itoa(int(m.HeapSys)))
+	s.registerGauge("runtime.usedMemory", strconv.Itoa(int(m.HeapInuse)))
```

Comment:
what is the plan for the missing metrics?

Reply:
I dont think there is any plan for it that I know of.

We will probably end up including some source code of a third party package in vendor folder.